pkg/getty/Dockerfile

```diff
@@ -17,4 +17,4 @@ COPY --from=mirror /out/ /
 COPY usr/ /usr/
 COPY etc/ /etc/
 CMD ["/usr/bin/rungetty.sh"]
-LABEL org.mobyproject.config='{"pid": "host", "net":"host", "binds": ["/run:/run", "/etc:/hostroot/etc","/tmp/ctr:/tmp/ctr", "/usr/bin/ctr:/usr/bin/ctr", "/usr/bin/runc:/usr/bin/runc", "/usr/bin/dist:/usr/bin/dist", "/var:/var","/containers:/containers","/dev:/dev","/sys:/sys"], "capabilities": ["all"]}'
+LABEL org.mobyproject.config='{"pid": "host", "net":"host", "binds": ["/run:/run", "/etc:/hostroot/etc", "/usr/bin/ctr:/usr/bin/ctr", "/usr/bin/runc:/usr/bin/runc", "/usr/bin/dist:/usr/bin/dist", "/var:/var","/containers:/containers","/dev:/dev","/sys:/sys"], "capabilities": ["all"]}'
```

Comment:
`/tmp/ctr` a) is not created is you aren't using `ctr` (which is the purpose of the tool) and b) isn't created even by newer versions of `ctr`.

The only thing it is/was used for AFAICT is to hold the stdio fifos from `ctr run` and `ctr exec`. There is/was nothing in there which is useful to any independent future run of `ctr`, you can run `ctr` in the `getty` container just fine without sharing `/tmp/ctr` with the host mount namespace.